misc/FIELD260FPS.py

```diff
@@ -1,420 +1,382 @@
-                        
-                            
-                        #chatGPT
-                        #byte_4= (int(rotZ/4)&0x3)|(  (int(rotX/4)&0xc) ) |( (int(rotY/4)&0x30))
-                        #byte_1= int(rotZ/4)& ~(0x3<<8)
-                        #byte_2= int(rotX/4)& ~(0xC<<6)
-                        #byte_3= int(rotY/4)& ~(0x30<<4)
-                        
-                        '''new interpretation, with b1b1b1b1b1b1b1b1-b2b2b2b2b2b2b2b2-b3b3b3b3b3b3b3b3-b4b4b4b4b4b4b4b4 = \
-                                                    RzRzRzRzRzRzRzRz-RxRxRxRxRxRxRxRx-RyRyRyRyRyRyRyRy-RzRzRxRxRyRy 0 0'''
-                        '''byte_4= ((rotZ&0x300)>>8) |((rotX&0x300)>>6) |   ((rotY&0x300)>>4)
-                        byte_1= rotZ&0x7F
-                        byte_2= rotX&0x7F
-                        byte_3= rotY&0x7F'''
-                        
-                        byte_4= ((rotZ>>8)&3) |(  ((rotX>>8)&3)<<2   ) |(   ((rotY>>8)&3)<<4 )
-                        byte_1= rotZ&0xFF
-                        byte_2= rotX&0xFF
-                        byte_3= rotY&0xFF
-                        
-                        
-                        
-                        outputfile.write(byte_1.to_bytes(1,'little'))
-                        outputfile.write(byte_2.to_bytes(1,'little'))
-                        outputfile.write(byte_3.to_bytes(1,'little'))
-                        outputfile.write(byte_4.to_bytes(1,'little'))
-            
-            outputfile.read( (6 + 4 *boneCount) )#read last frame
-        
-                
-        # Add around 2000 zeroes between characters, and rewrite model addresse and size              
-        outputfile.read()#end of file
-        outputfile.write(b'\x00'*2000)
-        New_size=outputfile.tell()-(New_address+4)
-      
-       
-        
-        
-        outputfile.seek(Address_in_file[charID]+4,0)
-        outputfile.write(New_size.to_bytes(4,'little'))
-        outputfile.write(New_size.to_bytes(4,'little'))#written twice
-        
-        outputfile.read()#end of file
-        #change address for next model
-        if charID<charCount-1:
-            New_address=outputfile.tell()-4
-            
-            outputfile.seek(Address_in_file[charID+1],0)
-            
-            outputfile.write(New_address.to_bytes(4,'little'))
-            outputfile.read()# back to end of file
-        
-    print("{} interpolated from 30fps to 60 fps !\n".format(filename))                      
-    inputfile.close()
-    outputfile.close()
-          
-          
-                
-    return
-
-
-    
-if __name__=="__main__":
-    import os,os.path,tkinter
-    os.system("cls") 
-    from tkinter import filedialog
-    from tkinter import *
-    root = Tk()
-    root.withdraw()
-    folder_selected = filedialog.askdirectory()
-
-    FIELD_TO_60FPS(folder_selected)
-
-
-
-            
-
-
-
-
-
-
-
-
+"""***********************************************
```

Comment:
```suggestion
#*****************************************************************************#
#    Copyright (C) 2024 Shunsq                                                #
#                                                                             #
#    This file is part of FF8 MCH                                             #
#                                                                             #
#    FF8 MCH is free software: you can redistribute it and/or modify          #
#    it under the terms of the GNU General Public License as published by     #
#    the Free Software Foundation, either version 3 of the License            #
#                                                                             #
#    FF8 MCH is distributed in the hope that it will be useful,               #
#    but WITHOUT ANY WARRANTY; without even the implied warranty of           #
#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the            #
#    GNU General Public License for more details.                             #
#*****************************************************************************#

###
# PLEASE NOTE:
# THIS IS A STANDALONE SCRIPT. YOU CAN LAUNCH IT USING ONLY PYTHON
# YOU DO NOT NEED BLENDER FOR IT!
###

"""***********************************************
```